<ui version="4.0" >
 <class>qtLineWidget</class>
 <widget class="QWidget" name="qtLineWidget" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>232</width>
    <height>196</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" >
   <property name="spacing" >
    <number>6</number>
   </property>
   <property name="leftMargin" >
    <number>0</number>
   </property>
   <property name="topMargin" >
    <number>0</number>
   </property>
   <property name="rightMargin" >
    <number>0</number>
   </property>
   <property name="bottomMargin" >
    <number>0</number>
   </property>
   <item>
    <layout class="QGridLayout" >
     <property name="leftMargin" >
      <number>0</number>
     </property>
     <property name="topMargin" >
      <number>0</number>
     </property>
     <property name="rightMargin" >
      <number>0</number>
     </property>
     <property name="bottomMargin" >
      <number>0</number>
     </property>
     <property name="horizontalSpacing" >
      <number>6</number>
     </property>
     <property name="verticalSpacing" >
      <number>6</number>
     </property>
     <item row="0" column="0" >
      <widget class="QCheckBox" name="visible" >
       <property name="text" >
        <string>Show Line</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1" >
      <widget class="QCheckBox" name="pickMeshPoint" >
       <property name="text" >
        <string>Snap On Mesh Point</string>
       </property>
       <property name="toolTip">
         <string>If checked, picked points using 'P' will be snapped on mesh points</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0" >
      <widget class="QComboBox" name="pickPoint">
       <item>
         <property name="text" >
          <string>Pick Both Points</string>
         </property>
       </item>
       <item>
         <property name="text" >
          <string>Pick Point 1</string>
         </property>
       </item>
       <item>
         <property name="text" >
          <string>Pick Point 2</string>
         </property>
       </item>
       <property name="toolTip">
         <string>Allow to choose wich point to pick, for easier replacing</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QGridLayout" >
     <property name="leftMargin" >
      <number>0</number>
     </property>
     <property name="topMargin" >
      <number>0</number>
     </property>
     <property name="rightMargin" >
      <number>0</number>
     </property>
     <property name="bottomMargin" >
      <number>0</number>
     </property>
     <property name="horizontalSpacing" >
      <number>6</number>
     </property>
     <property name="verticalSpacing" >
      <number>6</number>
     </property>
     <item row="0" column="0" >
      <widget class="QLabel" name="labelPoint1" >
       <property name="text" >
        <string>Point 1</string>
       </property>
       <property name="wordWrap" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="3" >
      <widget class="pqLineEdit" name="point1Z" />
     </item>
     <item row="0" column="2" >
      <widget class="pqLineEdit" name="point1Y" />
     </item>
     <item row="0" column="1" >
      <widget class="pqLineEdit" name="point1X" />
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <property name="spacing" >
      <number>6</number>
     </property>
     <property name="leftMargin" >
      <number>0</number>
     </property>
     <property name="topMargin" >
      <number>0</number>
     </property>
     <property name="rightMargin" >
      <number>0</number>
     </property>
     <property name="bottomMargin" >
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="labelPoint2" >
       <property name="text" >
        <string>Point 2</string>
       </property>
       <property name="wordWrap" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="pqLineEdit" name="point2X" />
     </item>
     <item>
      <widget class="pqLineEdit" name="point2Y" />
     </item>
     <item>
      <widget class="pqLineEdit" name="point2Z" />
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="xAxis" >
     <property name="text" >
      <string>X Axis</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="yAxis" >
     <property name="text" >
      <string>Y Axis</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="zAxis" >
     <property name="text" >
      <string>Z Axis</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>pqLineEdit</class>
   <extends>QLineEdit</extends>
   <header>pqLineEdit.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>visible</tabstop>
  <tabstop>point1X</tabstop>
  <tabstop>point1Y</tabstop>
  <tabstop>point1Z</tabstop>
  <tabstop>point2X</tabstop>
  <tabstop>point2Y</tabstop>
  <tabstop>point2Z</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
